Removes stray obj/item Metastation

dieamond13 opened the pull request:
## About The Pull Request
Removes a stray obj/item from a random Metastation wall in Port Quarter Solars and adds a maplint so that this doesn't happen again.
Closes #77267
## Why It's Good For The Game
A obj/item in a random wall is not good and should as it is a bug, this fixes that.
## Changelog
:cl:
fix: removes a stray obj/item from random metastation wall
/:cl:

dieamond13 committed:
Update MetaStation.dmm

dieamond13 committed:
Revert "Update MetaStation.dmm"

This reverts commit d23e0809291dfefe8f9c6bf1d1cf6a2f8344fa52.

dieamond13 committed:
Removes obj/item from Metastation port quarter solar wall

Removes an obj/item from a random wall in Meta's port quarter solars.

san7890 commented:
would you mind creating a maplint to ensure this doesn't happen in the future? you can read more about making one here: https://github.com/tgstation/tgstation/tree/master/tools/maplint

thanks

dieamond13 commented:
> would you mind creating a maplint to ensure this doesn't happen in the future? you can read more about making one here: https://github.com/tgstation/tgstation/tree/master/tools/maplint . you'll likely need to use `=/obj/item`
> 
> thanks

would I do this by doing this in the lints folder?:
=/obj/item:
  banned: true

dieamond13 committed:
Adds maplint to make sure obj/item doesnt spawn inside of walls.

Adds a maplint that doesn't allow /obj/item to be inside of closed turfs such as walls so that this issue doesn't happen again.

dieamond13 commented:
Alright, I added a maplint so that it shouldn't spawn inside closed turfs again.
